tests/integration/features/file_traceability/20_skip_invalid_utf8/test.itest

```diff
@@ -0,0 +1,9 @@
+# file.bin was created with echo "00 08 00 00" | xxd -r -p > file.bin
+RUN: %strictdoc export %S --output-dir Output | filecheck %s --dump-input=fail
+CHECK: info: Skip tracing binary file
```

Comment:
There are 3 such lines expected, one for each invalid input file. I'd like to handle them more specific, but couldn't figure out yet how to use filecheck with either regex or directory substitution. Will try to figure out later - hints are also appreciated :)

Reply:
Ignore, nothing particular here. Just a typo on my side.